graphql/resolvers/users.js

```diff
@@ -1224,5 +1277,40 @@ module.exports = {
         return user;
       }
     },
+
+    async updateYears() {
+      var users = await User.find();
+      users.forEach(async function(user){
+        const email = user.email;
+        const currDate = new Date();
+        var updatedAt = currDate;
+        if(user.updatedAt) updatedAt = new Date(user.updatedAt);
+        const msPerDay = 1000*60*60*24;
+        const difference = Math.round((currDate - updatedAt) / msPerDay);
+
+        var year = user.year;
```

Comment:
can we move this variable up maybe? There's a lot of variables at the top so sort them alphabetically.